packages/_infra/src/config.ts

```diff
@@ -10,19 +10,24 @@ export interface BaseMapsConfig {
 
     /** Domain name for CloudFront to use */
     AlbPublicDns: string;
+
+    /** Public URL base that tiles are served from */
+    PublicUrlBase: string;
 }
 
 export const BaseMapsDevConfig: BaseMapsConfig = {
     CogBucket: 'basemaps-cog-test',
     Route53Zone: 'nonprod.basemaps.awsint.linz.govt.nz',
     AlbPublicDns: 'dev.int.tiles.basemaps.linz.govt.nz',
+    PublicUrlBase: 'https://dev.tiles.basemaps.linz.govt.nz',
```

Comment:
need to remove the `https` as some will be served via http

Reply:
How do we know the original protocol if it is altered by proxies

Reply:
TODO: we should move where this https protocol definition to somewhere else.